dgraphql/resolvers/trxcache.go

```diff
@@ -0,0 +1,77 @@
+//Implement Least Recently Used cache to check for duplication transaction ID
+
+package resolvers
+
+import (
+	"container/list"
+)
+
+type TrxCache struct {
+	capacity int
+	data     map[string]interface{}
```

Comment:
I don't think `map` is an appropriate structure, because in this case, their value are always `true`. For the max. capacity of 1000,000, sorted slice/list would be enough.
Also, `order` would better to be a list of `*sting`.

Reply:
Yeah, for this case, I specifically use a map for data since it is faster than an array when you want to update or check if the key exists.
For the `order` list, I updated it to a list of strings and removed `TrxEntry` struct.

Reply:
Also update the max capacity to 100,00

Reply:
If you ever wanted to use `map`, the type should be `map[string]bool`, which is in fact a set data structure.

Reply:
updated to `map[string]bool`